<?xml version="1.0" encoding="UTF-8"?>
<robot rpa="false" generated="20190123 10:38:15.389" generator="Robot 3.1.1 (Python 2.7.14 on win32)">
<suite source="C:\Users\Acer\Desktop\robot\sharing-test-RobotFarmeWork\start.robot" id="s1" name="Start">
<test id="s1-t1" name="Open Chrome Browser">
<kw name="Open Browser" library="Selenium2Library">
<doc>Opens a new browser instance to the given ``url``.</doc>
<arguments>
<arg>${url}</arg>
<arg>chrome</arg>
</arguments>
<msg timestamp="20190123 10:38:15.755" level="INFO">Opening browser 'chrome' to base url 'about:blank'.</msg>
<status status="PASS" endtime="20190123 10:38:18.757" starttime="20190123 10:38:15.754"></status>
</kw>
<status status="PASS" endtime="20190123 10:38:18.764" critical="yes" starttime="20190123 10:38:15.753"></status>
</test>
<kw type="teardown" name="Close Browser" library="Selenium2Library">
<doc>Closes the current browser.</doc>
<status status="PASS" endtime="20190123 10:38:21.552" starttime="20190123 10:38:18.769"></status>
</kw>
<status status="PASS" endtime="20190123 10:38:21.552" starttime="20190123 10:38:15.392"></status>
</suite>
<statistics>
<total>
<stat fail="0" pass="1">Critical Tests</stat>
<stat fail="0" pass="1">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat fail="0" id="s1" name="Start" pass="1">Start</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20190123 10:38:15.730" level="ERROR">Error in file 'C:\Users\Acer\Desktop\robot\sharing-test-RobotFarmeWork\start.robot': Importing test library 'ExtendedSelenium2Library' failed: ImportError: No module named ExtendedSelenium2Library
Traceback (most recent call last):
  None
PYTHONPATH:
  C:\Python27\Scripts\robot.exe
  C:\Windows\SYSTEM32\python27.zip
  c:\python27\DLLs
  c:\python27\lib
  c:\python27\lib\plat-win
  c:\python27\lib\lib-tk
  c:\python27
  c:\python27\lib\site-packages</msg>
</errors>
</robot>
